src/js/vic-v2/containers/ConfirmationPage.jsx

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import _get from 'lodash/get';
```

Comment:
We normally `import _ from 'lodash/fp';`. We use the webpack babel plugin that transforms those imports into ones like this that just import the specific functions being used.